strands_navfn/package.xml

```diff
@@ -0,0 +1,52 @@
+<package>
+    <name>strands_navfn</name>
+    <version>1.12.2</version>
```

Comment:
I was just about to say that, yes, remove it, and put something in the description that acknowledges this fact.